### DISCUSSION TITLE
How to implement in swift is there any demo link or documentation for that?

### DISCUSSION THREAD

**Discussion by kupix** [bungee-audio-stretch/bungee#10] [Integrations]
Originally asked as https://github.com/kupix/bungee/issues/9 by @ktech643

**Comment by kupix** [comment]
Users of the closed-source Bungee Pro library have successfully integrated with Swift and since Pro uses the same API there is no reason why it shouldn't be possible with this project.

It may be necessary to build Bungee as a "framework" for MacOS / iOS: this is something we can add to CI if it turns out to be helpful.

**Reply by ktech643** [reply]
Thank you. Can you guide me how to make .xcframework from this library or what is the work around or any reference tutor I will create xcframework and do merge request.

**Reply by ktech643** [reply]
How to get Bungee Pro and what is the price for that

**Comment by ktech643** [comment]
Can anyone guide me for this

**Reply by kupix** [reply]
This github project builds a release that include Apple frameworks for iOS and MacOS. If you need an xcframework, it should be possible to create one from this or have cmake do that as part of the Bungee release process.

If you make progress with this and can improve the build process, please consider making a pull request.
